src/@chakra-ui/gatsby-plugin/components/Select.ts

```diff
@@ -0,0 +1,68 @@
+import { selectAnatomy } from "@chakra-ui/anatomy"
+import { createMultiStyleConfigHelpers, cssVar } from "@chakra-ui/react"
+import { defineMergeStyles, selectDefaultTheme } from "./components.utils"
+
+const { defineMultiStyleConfig, definePartsStyle } =
+  createMultiStyleConfigHelpers(selectAnatomy.keys)
+
+const $borderBaseWidth = cssVar("border-base-width")
+
+const baseStyleField = defineMergeStyles(selectDefaultTheme.baseStyle?.field, {
+  [$borderBaseWidth.variable]: "1px",
+  borderColor: "currentColor",
+  cursor: "pointer",
+  py: 2,
+  ps: 2,
+  pe: 8,
+  _active: {
+    bg: "primaryLowContrast",
+    boxShadow: "none",
+  },
+  _hover: {
+    boxShadow: "buttonHover",
+    color: "primary",
+  },
+  _focusVisible: {
```

Comment:
For some reason, this focus is not working 100% as I would expect. Same happens to the buttons.

When you click on them, we are showing these focus styles, and I wouldn't expect that to happen. Let me show you:

https://user-images.githubusercontent.com/468158/234983124-f1aa49fc-e721-4a98-89a8-f9c2212348fc.mp4

I would not expect to have the thick border on it when you click on it. Same with the button.

Perhaps we need to override the `_focus: null` here as well. What do you think?

Reply:
Interesting! This might be browser specific, because this is not an issue in Firefox.

In Chrome, this is coming from `focus-visible`, not `focus`. Might need to find an override for Chromium browsers?

Reply:
Uhmm...I'd expect it to work as in the Chakra docs examples. These [buttons](https://chakra-ui.com/docs/components/button/usage) don't display the focus border when I click them. And yes, if you look at the [theme file](https://github.com/chakra-ui/chakra-ui/blob/main/packages/components/theme/src/components/button.ts#LL11C10-L11C10) they are using the `focusVisible` event as well 🤔 that is why I was thinking on overriding the `focus` event to check if that fixed the issue.

Reply:
@pettinarip 
Hmmm I think it is a difference in pseudo state behavior. With a button, you would get the similar behavior with the focus styling on click if styling was applied to `:focus` and not `:focus-visible`. This is consistent between Firefox and Chrome.

I would not expect an inconsistency in browser heuristic here with `:focus-visible`.

But check out [W3 Specs for :focus-visible](https://www.w3.org/TR/selectors-4/#the-focus-visible-pseudo)

And specifically the bullet point for when the User Agent may or may not indicate focus with `:focus-visible`:

`If the user interacts with the page via a pointing device (mouse, touchscreen, etc.) and the focused element does not support keyboard input, don’t indicate focus`
(Maybe??)